Code/Sideloaded-TextTracks-Examples/Cocoapod/SideloadedTextTracksExample/ViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ViewController.swift
+//  SideloadedTextTracksExample
+//
+//  Created by Wonne Joosen on 13/04/2024.
+//
+
+import UIKit
+import THEOplayerSDK
+import THEOplayerConnectorSideloadedSubtitle
+
+class ViewController: UIViewController {
+    
+    
+    @IBOutlet weak var playerViewContainer: UIView!
+    var player: THEOplayer
+    
+    required init?(coder: NSCoder) {
+//        self.player = THEOplayer(with: nil, configuration: THEOplayerConfiguration(chromeless: false, ads: AdsConfiguration(
+//            showCountdown: true,
+//            preload: AdPreloadType.MIDROLL_AND_POSTROLL,
+//            googleIma: GoogleIMAAdsConfiguration(useNativeIma: false)
+//        )))
+        let playerConfiguration = THEOplayerConfigurationBuilder()
+        playerConfiguration.license = "<your license>"
+        self.player = THEOplayer(with: nil, configuration: playerConfiguration.build())
+        super.init(coder: coder)
+    }
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        
+        let playerView = PlayerView(player: player)
+        playerView.translatesAutoresizingMaskIntoConstraints = true
+        playerView.frame = playerViewContainer.bounds
+        playerView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+        playerViewContainer.addSubview(playerView)
+        
+    }
+    
+    
+    @IBAction func srtButtonClicked(_ sender: UIButton) {
+        print("[SideloadedTextTracksExample] Setting source with SRT subtitles")
+        player.setSourceWithSubtitles(source: SourceDescription(
+            source: TypedSource(src: "https://cdn.theoplayer.com/video/elephants-dream/playlist-no-subtitles.m3u8", type: "application/x-mpegurl"),
+            textTracks: [TextTrackDescription(src: "https://cdn.theoplayer.com/video/elephants-dream/captions.nl.srt", srclang: "en", isDefault: false, kind: .subtitles, label: "English", format: .SRT)]
```

Comment:
I suggest we use the `SSTextTrackDescription` type instead of `TextTrackDescription` since it would help point developers towards the many advantages that this connector offers (same for `vttButtonClicked`).

Reply:
✅